marklogic-data-hub-central/ui/e2e/cypress/e2e/monitor/monitorTableValidations.cy.tsx

```diff
@@ -47,79 +57,88 @@ describe("Monitor Tile", () => {
     cy.clearAllLocalStorage();
   });
 
-  let firstPageTableCellsJobId: any[] = [];
-  let firstPageTableCellsJobIdAux: any[] = [];
-  let firstPageTableCellsStepName: any[] = [];
-  let firstPageTableCellsStepType: any[] = [];
-  let firstPageTableCellsStatus: any[] = [];
-  let firstPageTableCellsEntityType: any[] = [];
-  let firstPageTableCellsDateTime: any[] = [];
-  let firstPageTableCellsStepName1: any[] = [];
-  let firstPageTableCellsStepType1: any[] = [];
-  let firstPageTableCellsStatus1: any[] = [];
-  let firstPageTableCellsEntityType1: any[] = [];
-  let firstPageTableCellsDateTime1: any[] = [];
-  let orginalDateTimeArr: any[] = [];
-  const flowName = "testMonitor";
-
-  it("Validate column order for Step Name,	Step Type,	StatusEntity, Type Start, Date and Time part 1", () => {
-    cy.log("**expand table and get data column of JobId**");
+  it("Ascending order validations Validate column order for Step Name,	Step Type,	StatusEntity, Type Start, Date and Time", () => {
```

Comment:
Text is better than previous. Any reason why we kept the extra spaces in the title?